libs/components/data-manager/src/lib/modules/data-manager/data-manager-column-picker/data-manager-column-picker.component.scss

```diff
@@ -3,3 +3,7 @@
 .sky-data-manager-column-picker-content sky-repeater-item-content {
   margin-top: $sky-margin-inline-compact;
 }
+
+.sky-data-manager-column-picker-footer-stacked-indicator {
+  margin: 0 $sky-margin-stacked-compact $sky-margin-stacked-default 0px;
```

Comment:
You've still got a `0px` hanging out here.

Reply:
Also, I would think you'd use one of the `$sky-margin-inline-` variables for the right margin instead of `$sky-margin-stacked-` which is for top/bottom margins.

Reply:
fixed in commit: [correct styles](https://github.com/blackbaud/skyux/pull/349/commits/5f739edfa7c29a842e8c586138458f7c64b3d4ec)